### ISSUE TITLE
Problem: Разделение имен на два предложения

### ISSUE THREAD

**Issue by voddan** [olegcherr/Reedy-for-Chrome#21]
Когда reedy встречает англоязычное имя типа `Mr. Bond`, она воспринимает его не только как два разных слова, но и как два разных предложения. 

В результате возникает пауза после `Mr.`, хотя имя должно показываться на одной строке с выделением на значащей части: `Mr. B_o_nd`.

В нынешнем состоянии очень сложно читать политические и научные тексты из=за постоянных остановок

**Comment by olegcherr** [comment]
Да, это есть в планах. В скором времени займёмся переработкой парсера.

**Comment by VasiliyRyabtsev** [comment]
Хочу добавить пожелание по парсеру примерно на ту же тему -  объединять предлоги, частицы, и, может быть, союзы.

В текстах встречаются последовательности коротких слов, которые показываются по отдельности, хотя их было бы логично склеивать и показывать одновременно.
"Было бы", как пример, "по мере", "в течение", "не самый", "и поэтому", "как только" и т.п.

Полезно, наверное, объединять и более длинные сочетания, привычные читателю (если, конечно, влезают по длине).
Если встретится фраза типа "как бы то ни было", то, возможно, её можно было бы показывать всю целиком, а сейчас она разбивается на пять слов, и внимание спотыкается.

Более того, возможно, есть смысл собрать в оффлайне статистику по встречающимся вместе словам (для разных языков), и объединять их - если слова в сочетании встречаются часто, то и склеенные должны хорошо восприниматься.

Из такого
```
— Мне кажется, оливковое масло гораздо лучше, — заметил юноша, и Аристотель не мог не согласиться.
```
тогда склеятся на одновременный показ
```Мне кажется```
```оливковое масло```
```не мог```
```не согласиться```


А из
```
Если бы я захотел захватить Афины, то уже сделал бы это. Но я ограничился тем, что утвердил свою власть в области непосредственного влияния Македонии.
```
получатся
```Если бы```
```я захотел```
```сделал бы```
```в области```
